components/brave_wallet_ui/common/slices/api.slice.ts

```diff
@@ -172,7 +172,8 @@ export function createWalletApi (
       'TokenSpotPrice',
       'TransactionsForAccount',
       'UserBlockchainTokens',
-      'WalletInfo'
+      'WalletInfo',
+      'RecipientByteCodes'
```

Comment:
Don't think we need a cache tag for these since the results for an address should not change

Reply:
Removed!